pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -18,6 +18,7 @@ limitations under the License.
 package mgr
 
 import (
+	srderrors "errors"
```

Comment:
Why not just have `errors`? What is `srderrors`?
```suggestion
	"errors"
```